msgpack-core/src/main/java/org/msgpack/core/MessagePack.java

```diff
@@ -38,6 +38,12 @@
     public static final Charset UTF8 = Charset.forName("UTF-8");
 
     /**
+     * Default packer/unpacker configurations
+     */
+    public static final PackerConfig DEFAULT_PACKER_CONFIG = new PackerConfig();
+    public static final UnpackerConfig DEFAULT_UNPACKER_CONFIG = new UnpackerConfig();
```

Comment:
I'm using those constants in MessagePackFactory instead of creating default packer config each time. We can either leave it public, or create another constant in MessagePackFactory to hold single instance. WDYT?